dist/stat_data.py

```diff
@@ -327,6 +332,22 @@ def __init__(self, name, desc, flags=''):
     CapacityStat('fsync_all_fh_total', 'background fsync file handles considered'),
     CapacityStat('fsync_all_time', 'background fsync time (msecs)', 'no_clear,no_scale'),
 
+    ##########################################
+    # Chunk cache statistics
+    ##########################################
+    ChunkCacheStat('chunk_cache_bytes', 'total bytes'),
+    ChunkCacheStat('chunk_cache_chunks', 'total chunks'),
+    ChunkCacheStat('chunk_cache_chunks_evicted', 'evicted chunks'),
+    ChunkCacheStat('chunk_cache_chunks_invalidated', 'removed chunks on becoming invalid'),
+    ChunkCacheStat('chunk_cache_exceeded_capacity', 'could not allocate due to exceeding capacity'),
+    ChunkCacheStat('chunk_cache_hits', 'number of hits'),
+    ChunkCacheStat('chunk_cache_io_failed', 'number of times a read from storage failed'),
+    ChunkCacheStat('chunk_cache_lookups', 'lookups'),
```

Comment:
In the block cache stats there is also a "misses" stat so that the user doesn't have to compute that. Do we want a `chunk_cache_misses` stat also?

Reply:
Change from keeping track of hits to keeping track of misses. Agree that misses is a more interesting stat.